schemas/WebpackOptions.json

```diff
@@ -1638,13 +1638,14 @@
               "properties": {
                 "host": {
                   "description": "A host.",
-                  "type": "number"
+                  "type": "string"
                 },
                 "port": {
                   "description": "A port.",
                   "type": "number"
                 }
               },
+              "required": ["host", "port"],
```

Comment:
both should be required https://github.com/webpack/webpack/blob/main/lib/hmr/lazyCompilationBackend.js#L44

Reply:
Can't you omit them? Without port is chooses a random port?

Reply:
```
TypeError: The argument 'options' must have the property "port" or "path". Received { host: '127.0.0.1' }
```

Reply:
I think random port is chosen without `options`

Reply:
looks like I'm right..
error from here https://github.com/nodejs/node/blob/master/lib/net.js#L1505
works fine without args https://github.com/nodejs/node/blob/master/lib/net.js#L1451
we can try to pass `port=0` https://github.com/nodejs/node/blob/master/lib/net.js#L1454